gcore/gdalrasterband.cpp

```diff
@@ -6306,9 +6308,10 @@ static inline double GetPixelValue(GDALDataType eDataType, bool bSignedByte,
             break;
         case GDT_Float16:
         {
+            using namespace std;
```

Comment:
I certainly don't claim this is wrong, but I wonder why there are two instances of `using namespace std;` rather than `std::`, as the latter seems narrower/safer and seems to be the prevailing idiom.

Reply:
> I certainly don't claim this is wrong, but I wonder why there are two instances of `using namespace std;` rather than `std::`, as the latter seems narrower/safer and seems to be the prevailing idiom.

there's some subtelty here I don't fully understand myself, but without that using namespace things don't build for me. Maybe @eschnett has a bit more clues